config/git/config

```diff
@@ -0,0 +1,2 @@
+[include]
```

Comment:
I think let's just expose all the settings directly to the user. These are often fairly personal, so makes sense that they're easy to change individually.